.github/workflows/test.yml

```diff
@@ -56,6 +56,16 @@ jobs:
           name: debug-files
           path: debug_files.tar.gz
           if-no-files-found: ignore
+      - name: Convert coverage data
+        run: |
+          nix profile install ./nix#go_1_20
+          go tool covdata textfmt -i=integration_tests/coverage -o profile.txt
```

Comment:
we has been building v1.0.x using go 1.19.x, which don't support the `-cover` feature I think.